.gitmodules

```diff
@@ -28,6 +25,9 @@
 [submodule "lib/libjpeg/libjpeg-turbo"]
 	path = lib/libjpeg/libjpeg-turbo
 	url = https://github.com/libjpeg-turbo/libjpeg-turbo.git
-[submodule "lib/sdl/SDL_image"]
+[submodule "lib/sdl/SDL_ttf"]
+	path = lib/sdl/SDL_ttf
+	url = https://github.com/libsdl-org/SDL_ttf.git
+[submodule "lib/sdl/SDL2_image"]
 	path = lib/sdl/SDL2_image
-	url = https://github.com/SDL-mirror/SDL_image.git
+	url = https://github.com/libsdl-org/SDL_image.git
```

Comment:
For easier review:

- Old: https://github.com/SDL-mirror/SDL_image/tree/b4f28c1fc9ab2f91ae594fc3e86622c5c13ffbbb
- New: https://github.com/libsdl-org/SDL_image/tree/ab2a9c602623193d61827ccd395302d92d90fc38

Looks good.

Worth noting: this is version https://github.com/libsdl-org/SDL_image/tree/release-2.0.5
This is the latest version available, so no further action is necessary.